modules/manifold/src/components/manifold.js

```diff
@@ -174,161 +181,156 @@ class Manifold extends PureComponent {
-                key="headline"
-                gridArea="headline"
-                isShown={!showBoth}
-                borderBottom
-                padded
-                flexDirection="row"
-              >
-                <Headline
-                  headers={[VIEW_NAME.PERF, VIEW_NAME.FEATURE]}
-                  onTabChange={this._toggleViewTab}
-                  isCoordinated={showBoth}
-                  showHelp={helpType === HELP_TYPE.PERF}
-                  onClickSplit={this._toggleViewMode}
-                  onClickHelp={() =>
-                    this._toggleHelpType(
-                      showLeftView ? HELP_TYPE.PERF : HELP_TYPE.FEATURE
-                    )
-                  }
-                  themeColor={THEME_COLOR}
-                />
-              </Panel>
-              <Panel
-                key="headline1"
-                gridArea="headline1"
-                isShown={showBoth}
-                borderBottom
-                borderRight
-                padded
-                flexDirection="row"
-              >
-                <Headline
-                  headers={[VIEW_NAME.PERF]}
-                  themeColor={THEME_COLOR}
-                  isCoordinated={showBoth}
-                  showHelp={helpType === HELP_TYPE.PERF}
-                  onClickSplit={this._toggleViewMode}
-                  onClickHelp={() => this._toggleHelpType(HELP_TYPE.PERF)}
-                />
-              </Panel>
-              <Panel
-                key="headline2"
-                gridArea="headline2"
-                isShown={showBoth}
-                borderBottom
-                flexDirection="row"
-              >
-                <Headline
-                  headers={[VIEW_NAME.FEATURE]}
-                  themeColor={THEME_COLOR}
-                  isCoordinated={showBoth}
-                  showHelp={helpType === HELP_TYPE.FEATURE}
-                  onClickSplit={this._toggleViewMode}
-                  onClickHelp={() => this._toggleHelpType(HELP_TYPE.FEATURE)}
-                />
-              </Panel>
+      // <StyletronProvider value={engine}>
+      <BaseProvider theme={theme || THEME}>
+        <Container width={width} height={height}>
+          <Content viewMode={viewMode}>
+            <Panel
+              key="headline"
+              gridArea="headline"
+              isShown={!showBoth}
+              borderBottom
+              padded
+              flexDirection="row"
+            >
+              <Headline
+                headers={[VIEW_NAME.PERF, VIEW_NAME.FEATURE]}
+                onTabChange={this._toggleViewTab}
+                isCoordinated={showBoth}
+                showHelp={helpType === HELP_TYPE.PERF}
+                onClickSplit={this._toggleViewMode}
+                onClickHelp={() =>
+                  this._toggleHelpType(
+                    showLeftView ? HELP_TYPE.PERF : HELP_TYPE.FEATURE
+                  )
+                }
+                themeColor={THEME_COLOR}
```

Comment:
is `THEME_COLOR` something we could just get from the theme provider in the future? instead of having to pass it as a prop

Reply:
This is a good point